### ISSUE TITLE
Loading images in css

### ISSUE THREAD

**Issue by LestaD** [webpack-contrib/style-loader#93]
I try:

``` css
# ./style.css
.searchBg {
  background-image: url(sprite/searchBg.jpg)
}
```

``` js
# ./component.js
import css from './style.css';

// begin component
  <div className={css.searchBg} />
// end component
```

What i see in chrome:
![image](https://cloud.githubusercontent.com/assets/5620073/11242182/32542fdc-8e11-11e5-8b93-3563b09b5abb.png)

But image not loading to background.
When i click on "Open Link in new tab", link opens as:

![image](https://cloud.githubusercontent.com/assets/5620073/11242207/58689ff0-8e11-11e5-9954-77ec78af2c74.png)

**Comment by Strate** [comment]
From README:

> Note about source maps support and assets referenced with url: when style loader is used with ?sourceMap option, the CSS modules will be generated as Blobs, so relative paths don't work (they would be relative to chrome:blob or chrome:devtools). In order for assets to maintain correct paths setting output.publicPath property of webpack configuration must be set, so that absolute paths are generated.

Btw, why style-loader can not replace relative urls to absolute urls by itself?

**Comment by tizmagik** [comment]
See https://github.com/webpack/style-loader/issues/55 for some workarounds/solutions to this issue.

**Comment by michael-ciniawsky** [comment]
Will be fixed by either #96 or #124
